### PR TITLE
Native image update

### LINKED ISSUE #2076: Remove internal GraalVM API use
We currently use internal APIs of GraalVM in our extensions.
We will need to replace these with our own classpath scanning mechanism.

### PR DESCRIPTION
**Pull request by tomas-langer**
* Remove use of AutomaticFeature (unsupported API)
* Remove use of context impl (unsupported API)
* Using a third-party classpath scanning tool

Resolves #2076 

Signed-off-by: Tomas Langer <tomas.langer@oracle.com>

### CODE REVIEW

**Comment by tomas-langer** [comment]
Updated to latest classgraph version, as it resolved a problem I reported.
https://github.com/classgraph/classgraph/releases/tag/classgraph-4.8.87

File: integrations/graal/native-image-extension/src/main/java/io/helidon/integrations/graal/nativeimage/extension/HelidonReflectionFeature.java [hunk 1]
```diff
@@ -120,83 +128,135 @@ public void beforeCompilation(BeforeCompilationAccess access) {
...
+                .stream()
+                .map(classInfo -> {
+                    Class<?> clazz = null;
+                    try {
+                        clazz = classInfo.loadClass();
+                    } catch (Exception e) {
+                        traceParsing(() -> "Class " + classInfo.getName() + " annotated by " + annotation + " cannot be loaded");
+                        if (TRACE_PARSING) {
+                            traceParsing(() -> "\tException class: " + e.getClass().getName() + ", message: " + e.getMessage());
+                        }
```
Comment (danielkec) [inline integrations/graal/native-image-extension/src/main/java/io/helidon/integrations/graal/nativeimage/extension/HelidonReflectionFeature.java:176]: `if (TRACE_PARSING)` is already inside of  `traceParsing`

Comment (tomas-langer) [reply integrations/graal/native-image-extension/src/main/java/io/helidon/integrations/graal/nativeimage/extension/HelidonReflectionFeature.java:176]: changed from Exception.printStackTrace, forgot to clear up. Will do.

Comment (tomas-langer) [reply integrations/graal/native-image-extension/src/main/java/io/helidon/integrations/graal/nativeimage/extension/HelidonReflectionFeature.java]: Fixed

**Pull request merged**